tests/testthat/test-backend-common.R

```diff
@@ -1,144 +1,17 @@
-  # by_samples <- audiofile_loader(filepath_wav, offset = 1, duration = 40000, unit = "samples")
-  # expect_equal(length(by_samples$waveform[[1]]), 40000)
-  # expect_equal(class(by_samples), c("audiofile", "list"))
-
-  # MP3
-  # expect_error(by_samples <- audiofile_loader(filepath_mp3), class = "runtime_error")
-})
-
-test_that("tuneR_loader works", {
-
-  a <- torchaudio:::tuneR_read_mp3_or_wav(filepath_mp3, from = 1, to = 40001, unit = "samples")
-  # MP3
-  by_samples <- tuneR_loader(filepath_mp3, offset = 1, duration = 40000, unit = "samples")
-  expect_equal(length(by_samples@left), 40000)
-  expect_equal(class(by_samples)[1], c("Wave"))
-
-  by_samples <- tuneR_loader(filepath_mp3, offset = 1, duration = 100, unit = "samples")
-  expect_equal(length(by_samples@left), 100)
-  expect_equal(class(by_samples)[1], c("Wave"))
-
-  by_time <-tuneR_loader(filepath_mp3, offset = 1, duration = 1, unit = "time")
-  expect_gt(length(by_time@left), 40000)
-  expect_lt(length(by_time@left), 50000)
-  expect_equal(class(by_time)[1], c("Wave"))
-
-  # WAV
-  by_samples <- tuneR_loader(filepath_wav, offset = 1, duration = 40000, unit = "samples")
-  expect_equal(length(by_samples@left), 40000)
-  expect_equal(class(by_samples)[1], c("Wave"))
-
-  by_samples <- tuneR_loader(filepath_wav, offset = 1, duration = 100, unit = "samples")
-  expect_equal(length(by_samples@left), 100)
-  expect_equal(class(by_samples)[1], c("Wave"))
-
-  by_time <-tuneR_loader(filepath_wav, offset = 1, duration = 1, unit = "seconds")
-  expect_gte(length(by_time@left), 8000)
-  expect_lte(length(by_time@left), 8000)
-  expect_equal(class(by_time)[1], c("Wave"))
-})
-
-
-test_that("transform_to_tensor works", {
-  # Wave
-  by_samples <- tuneR_loader(filepath_mp3, offset = 1, duration = 48000, unit = "samples")
-  waveform_and_sample_rate <- transform_to_tensor(by_samples)
-  expect_equal(waveform_and_sample_rate[[2]], sample_mp3@samp.rate)
-  expect_tensor(waveform_and_sample_rate[[1]])
-
-  # av
-  by_samples <- av_loader(filepath_mp3, offset = 1, duration = 48000, unit = "samples")
-  waveform_and_sample_rate <- transform_to_tensor(by_samples)
-  expect_equal(waveform_and_sample_rate[[2]], sample_mp3@samp.rate)
-  expect_tensor(waveform_and_sample_rate[[1]])
-
-  # audiofile
-  # by_samples <- audiofile_loader(filepath_wav, offset = 1, duration = 48000, unit = "samples")
-  # waveform_and_sample_rate <- transform_to_tensor(by_samples)
-  # expect_equal(waveform_and_sample_rate[[2]], sample_mp3@samp.rate)
-  # expect_tensor(waveform_and_sample_rate[[1]])
-})
-
-test_that("set_audio_backend and torchaudio_loader works", {
+test_that("set_audio_backend and torchaudio_load works", {
   loader <- getOption("torchaudio.loader")
   expect_equal(class(loader), "function")
 
   set_audio_backend(tuneR_loader)
-  audio <- torchaudio_loader(filepath_mp3, offset = 1, duration = 100, unit = "samples")
+  audio <- torchaudio_load(filepath_mp3, offset = 1, duration = 100, unit = "samples")
```

Comment:
We could use [patrick](https://github.com/google/patrick) to make it easier to re-use test code.

Reply:
that's great advice! I think I'll do that in a separate PR, though.